statvar_imports/brazil_visdata/FoodBasketDistribution/common_metadata.csv

```diff
@@ -0,0 +1,17 @@
+parameter,value
+url,
+description,Acquisition of Basic Baskets for families for Specific Population
+#place_type,
+#places_within,
+start_date,

```

Comment:
Let's add a reference to the places_csv
```
places_resolved_csv,../Brazil_Places_Resolved.csv
```

Reply:
Can we keep it at both places - Metadata and the in the command line?
places_resolved_csv does not get recognised when placed inside the metadata but it works fine when passed as a flag in the command.